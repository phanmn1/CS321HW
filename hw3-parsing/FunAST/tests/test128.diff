1c1
< Invalid syntax at (9,3), )
---
> Invalid syntax at (3,3), fact
