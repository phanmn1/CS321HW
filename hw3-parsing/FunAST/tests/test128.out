Invalid syntax at (3,3), fact
